.github/workflows/create-jurassic-site.yml

```
@@ -0,0 +1,327 @@
+            >> "$GITHUB_STEP_SUMMARY"
+
+      # -----------------------------------------------------------------------
+      # 1. Check out the selected Stripe branch and build the extension ZIP.
+      # -----------------------------------------------------------------------
+      - name: Checkout Stripe branch (${{ inputs.stripe-branch }})
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.stripe-branch }}
+
+      - name: Setup PHP and Node
+        uses: ./.github/actions/setup-env
```

actionlint (1.7.11)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/setup-env": line 5: unexpected key "type" for definition of input "php-version"

(action)